scripts/check_defusedxml_fallback.py

```
@@ -0,0 +1,269 @@
+    """True if the module imports any stdlib ``xml.*`` name anywhere."""
+    for node in ast.walk(tree):
+        if isinstance(node, ast.Import):
+            if any(_is_xml_module(alias.name) for alias in node.names):
+                return True
+        elif isinstance(node, ast.ImportFrom) and node.module is not None:
+            if _is_xml_module(node.module):
+                return True
+    return False
+
+
+def _try_triggers_fallback(node: ast.Try, module_xml_imports: bool) -> bool:
```

Ruff (0.15.13)

[warning] Boolean-typed positional argument in function definition

(FBT001)